File: Tendr/node_src/api/DataColumn.ts
export interface DataColumn {
    key: string;
    path: string;
    name: string;
    align?: "left" | "right" | "center",
    sortable: boolean;
    width?: number;
    urlProperty?: string;
}

export enum DataColumnAlign {
    Left,
    Right,
    Center
}

File: src/Montr.Core/Models/DataColumn.cs
namespace Montr.Core.Models
{
	// todo: rename to TableColumn (?)
	public class DataColumn
	{
		public string Key { get; set; }

		public string Path { get; set; }

		public string Name { get; set; }

		public string UrlProperty { get; set; }

		public DataColumnAlign Align { get; set; }

		public DataColumnFixed? Fixed { get; set; }

		public bool Sortable { get; set; }

		public SortOrder? DefaultSortOrder { get; set; }

		public short? Width { get; set; }
	}
}
